phoenix-core/src/it/java/org/apache/phoenix/end2end/MaxLookbackIT.java

```diff
@@ -213,51 +274,57 @@ public void testRecentMaxVersionsNotCompactedAway() throws Exception {
         String thirdValue = "ghi";
         try (Connection conn = DriverManager.getConnection(getUrl())) {
             String dataTableName = generateUniqueName();
-            String indexStem = generateUniqueName();
-            createTableAndIndexes(conn, dataTableName, indexStem, versions);
-            long afterInsertSCN = org.apache.phoenix.util.EnvironmentEdgeManager.currentTimeMillis();
+            String indexName = generateUniqueName();
+            createTable(dataTableName);
+            //increment by 10 min to make sure we don't "look back" past table creation
```

Comment:
Nit: the comment is not right (nothing here equals 10mins).. wait is only 1 ms now

Reply:
Thanks, good catch. Fixed.